mock/docs/site-defaults.cfg

```diff
@@ -419,9 +418,23 @@
 ## Important! You should register your host machine first!
 # config_opts['subscription-manager.conf'] = 'put file contents here.'
 ## This will only work with DNF and when repo is configured with modules=1 for repo in dnf.conf.
-## This is executed just before 'chroot_setup_cmd'.
-# config_opts['module_enable'] = ['list', 'of', 'modules']
-# config_opts['module_install'] = ['module1/profile', 'module2/profile']
+
+# List of module commands to be executed when initializing chroot, before
+# `chroot_setup_cmd`.  Each command is a pair like `(action, module_specs)`
+# where `module_specs` is a comma-separated list of module specifications.
+# The commands are executed in order they are configured here, and each
+# `action` can be executed multiple times.
+#
+## Artificial example: (a) Disable any potentially enabled postgresql module
+## stream, (b) enable _specific_ postgresql and ruby module streams,
+## (c) install the development nodejs profile and (d) disable it immediately.
+#config_opts['module_setup_commands'] = [
+#  ('disable', 'postgresql'),
+#  ('enable',  'postgresql:12, ruby:2.6'),
+#  ('install', 'nodejs:13/development'),
+#  ('disable', 'nodejs'),
+#]
```

Comment:
@dmach could you please quickly review that this is a reasonable mock chroot configuration design?